strapi/strapi
# Remove admin config file

### What does it do?

It removes the config.js file that was used to perform the following customisation in the admin:
- removal of the tutorial box
- disabled the api call to github to know the latest release

### Why is it needed?

In the v4 customisation won't be made using the extensions system in the admin. For these kind of settings it is better to work with env variables.

### How to test it?

In the example app add the following:

```
STRAPI_ADMIN_SHOW_TUTORIALS=false
STRAPI_ADMIN_UPDATE_NOTIFICATION=false
```

You should not see the tutorial video nor the update notification.



To be merged after #10237
